__tests__/app.test.js

```diff
@@ -126,4 +126,51 @@ describe("GET /api/articles", () => {
           expect(body.msg).toBe("Route Not Found")
       })
 });
-});
\ No newline at end of file
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+          expect(body).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 3
+        });
+      });
+  });
+ 
+ 
+  test("400: Responds with 'Bad Request' for invalid article id", () => {
+    return request(app)
+        .get("/api/articles/not-a-valid-id/comments")
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request");
+        });
+ });
+ 
+ test("404: Responds with 'Not Found' for non-existent article id", () => {
+    return request(app)
+        .get("/api/articles/999999/comments")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Not Found");
+        });
+ });
+
+ test("404: Responds with 'Not Found' for non-existent article id", () => {
+    return request(app)
+        .get("/api/articles/3/comment")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Route Not Found");
+        });
```

Comment:
This last test isn't really needed as the route will fall under the generic error handler you already set up. There is one test missing though! What if the article does exist but has no comments? For example article id 2. This should respond with the empty rows array. Please do add this test!